RC Security Reorg Phase 1: Access control section (#2491)
RedisLabs/redislabs-docs: content/rc/security/access-control/data-access-control/default-user.md

Use the copy button to copy the password to the clipboard:

{{<image filename="images/rc/button-database-password-copy.png"  alt="Use the Copy button to copy the default user password." >}}{{< /image >}}

You'll need to use this password whenever you connect to your database using a Redis client. For example,
in the Redis CLI, you use the AUTH command to provide this password:

```sh
AUTH 4kTtH2ddXfN2sFmXE6sowOLukxiaJhN8n
```

See your Redis client's documentation to learn how to provide your password when connecting.

## Change password

To change the default user password for your database:

1. From the database **Configuration** tab, select **Edit database**.

    {{<image filename="images/rc/button-database-edit.png" width="150px" alt="The Edit database button lets you change the database's default user password." >}}{{< /image >}}

1. Under the **Security** section, enter the new password in the **Default user password** field.

1. Select **Save database** to update the password.

    {{<image filename="images/rc/button-database-save.png" width="150px" alt="Use the Save database button to save the new password." >}}{{< /image >}}

## Turn off default user

> Once you have set up [Role-based access control]({{<relref "rc/security/access-control/data-access-control/role-based-access-control">}}) to control who can access your database, we recommend that you turn off default user access.

Reviewer: ```suggestion
After you set up [role-based access control]({{<relref "rc/security/access-control/data-access-control/role-based-access-control">}}) to limit who can access your database, we recommend that you turn off default user access.
```
